packages/autocomplete-js/src/createAutocompleteDom.ts

```diff
@@ -172,6 +174,7 @@ export function createAutocompleteDom<TItem extends BaseItem>({
     });
     const detachedCancelButton = createDomElement('button', {
       type: 'button',
+      title: 'Cancel',
```

Comment:
This should not be necessary, as screen readers will get their information from the cancel button's direct `textContent`. It is different for the search button which has an SVG inside.

```suggestion
```